Examples/OneSignalDemo/app/src/main/AndroidManifest.xml

```diff
@@ -18,11 +18,11 @@
         android:theme="@style/AppTheme"
         tools:ignore="GoogleAppIndexingWarning">
 
-        <meta-data android:name="com.onesignal.NotificationExtensionServiceClass"
-            android:value="com.onesignal.sdktest.notification.AppNotificationExtensionService" />
+        <meta-data android:name="com.onesignal.NotificationServiceExtension"
+            android:value="com.onesignal.sdktest.notification.AppRemoteNotificationExtensionService" />
```

Comment:
Value needs to be updated to `com.onesignal.sdktest.notification.NotificationServiceExtension` in this example since the class name changed in the example app.